packages/front-end/components/Experiment/PValueColumn.tsx

```diff
@@ -0,0 +1,127 @@
+import clsx from "clsx";
+import { FC } from "react";
+import { SnapshotMetric } from "back-end/types/experiment-snapshot";
+import { MetricInterface } from "back-end/types/metric";
+import { ExperimentStatus } from "back-end/types/experiment";
+import {
+  hasEnoughData,
+  isBelowMinChange,
+  isSuspiciousUplift,
+} from "@/services/experiments";
+import { useOrganizationMetricDefaults } from "@/hooks/useOrganizationMetricDefaults";
+import usePValueThreshold from "@/hooks/usePValueThreshold";
+import Tooltip from "../Tooltip/Tooltip";
+import NotEnoughData from "./NotEnoughData";
+
+const PValueColumn: FC<{
+  metric: MetricInterface;
+  status: ExperimentStatus;
+  isLatestPhase: boolean;
+  startDate: string;
+  snapshotDate: Date;
+  baseline: SnapshotMetric;
+  stats: SnapshotMetric;
+}> = ({
+  metric,
+  status,
+  isLatestPhase,
+  startDate,
+  snapshotDate,
+  baseline,
+  stats,
+}) => {
+  const {
+    getMinSampleSizeForMetric,
+    metricDefaults,
+  } = useOrganizationMetricDefaults();
+  const pValueThreshold = usePValueThreshold();
+  const minSampleSize = getMinSampleSizeForMetric(metric);
+  const suspiciousChange = isSuspiciousUplift(
+    baseline,
+    stats,
+    metric,
+    metricDefaults
+  );
+  const belowMinChange = isBelowMinChange(
+    baseline,
+    stats,
+    metric,
+    metricDefaults
+  );
+  const enoughData = hasEnoughData(baseline, stats, metric, metricDefaults);
+  const shouldHighlight =
+    metric &&
+    baseline?.value &&
+    stats?.value &&
+    enoughData &&
+    !suspiciousChange &&
+    !belowMinChange;
+  const expectedDirection = metric.inverse
+    ? stats.expected < 0
+    : stats.expected > 0;
+  const statSig = stats.pValue < pValueThreshold;
+
+  let sigText = "";
+  let className = "";
+
+  if (shouldHighlight && statSig && expectedDirection) {
+    sigText = `Significant win as the p-value is below ${pValueThreshold} and the change is in the desired direction.`;
+    className = "won";
+  } else if (shouldHighlight && statSig && !expectedDirection) {
+    sigText = `Significant loss as the p-value is below ${pValueThreshold} and the change is in the opposite direction.`;
```

Comment:
Maybe "and the change is opposite to the desired direction" or "and the change is _not_ in the desired direction"

Reply:
Sure, addressed
<img width="450" alt="image" src="https://user-images.githubusercontent.com/2374625/211399480-9cb12bcb-fd7e-4b43-96ad-37ba56069ee9.png">